docs/components/forms/index.md

```diff
@@ -0,0 +1,131 @@
+
+---
+
+title: Forms
+
+category: components
+
+slug: forms
+
+position: 3
+
+---
+
+
+
+# Forms
+
+
+
+The Form allows the user to provide information in a structured way.
+
+
+
+## Rendering
+
+
+
+```html
+
+<!-- default rendering -->
+
+<div  class="k-form k-form-md">
+	<div class="k-form-field">
+		<label class="k-label k-form-label">Label</label>
+		<div class="k-form-field-wrap">
+			<input class="k-input k-input-md k-rounded-md k-input-solid">
+		</div>
+	</div>
+	<div class="k-form-field">
+		<label class="k-label k-form-label">Label</label>
+		<div class="k-form-field-wrap">
+			<input class="k-input k-input-md k-rounded-md k-input-solid">
+		</div>
+	</div>
+	<div class="k-form-buttons">
+		<button type="submit">Submit</button>
+	</div>
+</div>
+
+```
+## States
+The form supports validation which shows an error message if the input is not valid.
+```html
+<div  class="k-form k-form-md">
+	<div class="k-form-field">
```

Comment:
add k-form-field-error class to the form-field. Remove k-text-error class from the k-label k-form-label, the correct error styling should come from k-form-field-error; after the input the correct element is:
`<span class="k-form-error k-invalid-msg">Please enter a valid email</span>`